web/resources.go

```diff
@@ -235,6 +235,37 @@ func getIRS() []dao.RunningService {
 	return services
 }
 
+func restGetServicesForMigration(w *rest.ResponseWriter, r *rest.Request, client *node.ControlClient) {
+	serviceID, err := url.QueryUnescape(r.PathParam("serviceId"))
+	if err != nil {
+		restBadRequest(w, err)
+		return
+	}
+	services := []service.Service{}
+	if err := client.GetServiceList(serviceID, &services); err != nil {
+		glog.Errorf("Could not get services: %v", err)
+		restServerError(w, err)
+		return
+	}
+	w.WriteJson(&services)
+}
+
+func restPostServicesForMigration(w *rest.ResponseWriter, r *rest.Request, client *node.ControlClient) {
+	var smr dao.ServiceMigrationRequest
+	err := r.DecodeJsonPayload(&smr)
+	if err != nil {
+		glog.Errorf("Could not decode services for migration: %v", err)
+		restServerError(w, err)
+		return
+	}
+	var unused int
+	if err = client.MigrateServices(smr, &unused); err != nil {
+		restServerError(w, err)
```

Comment:
Just FYI - both restBadRequest() and restServerError() return an HTTP 500 which is totally wrong.  500 is only correct for situations like the server segfaults.  But fixing all the REST methods to return the proper HTTP status codes is outside the scope of this PR